content/policies/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: npm Policies
+---
+
+These are updated from time to time.  Their sources are stored in a git
```

Comment:
This line should likely be updated to reference the documentation repo rather than the policies repo